hibernate-core/src/main/java/org/hibernate/annotations/Table.java

```diff
@@ -87,10 +87,8 @@
 	 * If enabled, Hibernate will insert a row only if the columns of the secondary table
 	 * would not all be null, and will always use an outer join to read the columns. Thus,
 	 * by default, Hibernate avoids creating a row of null values.
-	 * <p>
-	 * <em>Only applies to secondary tables.</em>
 	 *
-	 * @deprecated use {@link SecondaryRow#optional()}
+	 * @apiNote Only relevant for secondary tables
```

Comment:
Hey, what, I want `@Table` to go away, not resurrect it...

Reply:
Which Jira did you use to remove it?

Reply:
https://github.com/hibernate/hibernate-orm/commit/aef9ab24257d58c85d1ee58add0f19f66b11867a#diff-52e80acb0c8fe3afd4721dec7bac45e2cf6141d20d76a91752e26da7c1143376

^ that's the last commit before mine.  You haven't deleted it there.  Am I misunderstanding you?

Reply:
Oh the deprecation.  Yeah it was a brutal rebase; I'm sure I missed stuff.  Did not help I had to sift through stuff 
 like brand new mutation operations (delete no version) as I was working on how mutations work ;)

Reply:
I fixed it locally.  Not going to push here though as I'm just waiting for CI to finish before pushing this upstream

Reply:
yeah fine all good